### PR TITLE
Remove known issue about impersonation

### PR DESCRIPTION
**Pull request by awholegunch**

### CODE REVIEW

**Comment by awholegunch** [comment]
I found one more broken link to fix.

File: cdap-docs/admin-manual/source/security/impersonation.rst [hunk 1]
```diff
@@ -47,7 +47,3 @@ The configured HDFS delegation token timeout must be longer than the configured
 partition duration (``stream.partition.duration``), which has a default value of
 one hour (3600000). It must also be larger than the log saver's maximum file
 lifetime (``log.saver.max.file.lifetime.ms``), which has a value of six hours (21600000).
-
```
Comment (anew) [inline cdap-docs/admin-manual/source/security/impersonation.rst:50]: I believe there is one version of Hive (0.13, 0.14?) for which it still does not work

Comment (poornachandra) [reply cdap-docs/admin-manual/source/security/impersonation.rst:50]: Are you talking about this? https://issues.cask.co/browse/CDAP-8140

Comment (awholegunch) [reply cdap-docs/admin-manual/source/security/impersonation.rst:50]: Should we then change the note to instead say that per 8140, if you have Hive 0.13 and impersonation, cdap explore does not work?

Comment (poornachandra) [reply cdap-docs/admin-manual/source/security/impersonation.rst:50]: Yes

**Comment by anew** [comment]
LGTM

**Pull request merged**